### PR TITLE
Speed up some triangle math hotspots

### PR DESCRIPTION
**Pull request by mquander**
This was the lowest-hanging optimization fruit I found without really sacrificing any readability yet -- reduces tree construction time by roughly 10% on my Firefox.

### CODE REVIEW

**Comment by gkjohnson** [comment]
Thanks I'll take a better look at this later this evening but it looks good at first glance.

Regarding timing -- I put together a PR that adds timing and memory benchmarks so it's easier to consistently track and compare: https://github.com/gkjohnson/threejs-fast-raycast/pull/27. I'd appreciate your thoughts on it!

File: lib/GeometryUtilities.js [hunk 1]
```diff
@@ -46,66 +43,71 @@ const getLongestEdgeIndex = bb => {
...
+	let x, y, z;
 
 	for ( let i = 0, l = tris.length; i < l; i ++ ) {
 
 		const tri = tris[ i ];
 
 		for ( let v = 0; v < 3; v ++ ) {
 
-			const x = getValFunc( geo, tri, v, 0 );
-			const y = getValFunc( geo, tri, v, 1 );
```
Comment (gkjohnson) [inline lib/GeometryUtilities.js]: Is declaring new variables in a for loop body really a bottle neck? I didn't know that. Everything else looks good -- I'm just curious about this!

**Comment by gkjohnson** [comment]
Gonna merge this -- it looks good. Thanks!

File: lib/GeometryUtilities.js (hunk 1 above)
Comment (mquander) [reply lib/GeometryUtilities.js]: I don't think this part mattered, I think the part that mattered was updating a local variable directly vs. updating properties on the local vector.

**Pull request merged**